src/App/Range.cpp

```diff
@@ -37,7 +37,7 @@ using namespace App;
 const int App::CellAddress::MAX_ROWS = 16384;
 const int App::CellAddress::MAX_COLUMNS = 26 * 26 + 26;
 
-Range::Range(const char * range)
+Range::Range(const char * range, bool normalize)
```

Comment:
Per @wwmayer's various comments recently about "boolean blindness", it might be a good idea to make this an enum (or something along those lines) instead of a bool.

Reply:
But I thought he is referring to cases of multiple bool arguments? Does that mean we aren't supposed to use any bool as function argument, @wwmayer?

Reply:
I also understood it as @realthunder . A single bool does not harm in my opinion.

Reply:
If from the function name it's obvious what it does then a (single) boolean is perfectly allowed. E.g. for `setVisible(bool)` it is clear what the arguments `true` or `false` will do.

If the intention of a boolean is not clear and one has to look at the documentation in order to get its meaning then the boolean might be the wrong type. Here is an article about Qt design principles that addresses this issue too: https://wiki.qt.io/API_Design_Principles#Avoiding_Common_Traps

In the past we had some cases where functions accepted several boolean (in an extreme case up to six) which does not only make client code unreadable but also the implementation of that function.